README.md

```diff
@@ -14,45 +14,55 @@ The toolkit library jar is hosted on https://esri.jfrog.io/artifactory/arcgis.
 
 To add the dependency to your project using Gradle:
 ```groovy
-apply plugin: 'application'
-
-// Runtime SDK dependency
-apply plugin: 'com.esri.arcgisruntime.java'
-buildscript {
-    repositories {
-        maven {
-            url 'https://esri.jfrog.io/artifactory/arcgis'
-        }
-    }
-    dependencies {
-        classpath 'com.esri.arcgisruntime:gradle-arcgis-java-plugin:1.0.0'
-    }
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.8'
+}
+
+// Replace with version number of ArcGIS SDK you are using in your app, such as:
+// arcgisVersion = '100.11.0'. See table below for SDK Versions that support the toolkit.
+ext {
+  arcgisVersion = '100.2.1'
```

Comment:
You might as well update this to 100.11.0 too, so users are guided to the latest version.

Reply:
DONE